src/main/java/org/isf/utils/excel/ExcelExporter.java

```diff
@@ -438,12 +428,12 @@ public void exportResultsetToExcel(ResultSet resultSet, File exportFile) throws
 			}
 			workbook.write(fileStream);
 			fileStream.flush();
-			fileStream.close();
-
+		} catch (FileNotFoundException e) {
+			throw new OHException(e.getLocalizedMessage());
+		}
 		} catch (SQLException e) {
 			throw new OHException(MessageBundle.getMessage("angal.sql.problemsoccurredwiththesqlinstruction.msg"), e);
 		}
```

Comment:
there is something wrong here in the indentation

Reply:
reformatted the entire method